translations/nl-be.json

```diff
@@ -121,7 +121,8 @@
   "documents-amount": "Aantal documenten",
   "document-add": "Document toevoegen",
   "document-link-old": "Reeds bezorgde documenten koppelen",
-  "document-delete": "Document verwijderen",
+  "document-delete": "Verwijderen",
```

Comment:
Deze change zorgt voor falende testen, omdat die test nog kijkt naar tekst ipv een data-tag.
Maar dat wil ook zeggen dat deze changes een side-effect heeft in bestaande views.
Terwijl dit niet de bedoeling was

Reply:
gebruik op die plaats dan de vertaling voor {{t "delete"}} ipv deze aan te passen

Reply:
aah ok

Reply:
🤦🏻‍♀️

Reply:
aangepast